man/config

```diff
@@ -0,0 +1,11 @@
+Usage: releasify draft [--arg|-a <string>] [--value <string>] [--help|-h]
```

Comment:
Unfortunately storing values through CLI arguments defeat the purpose of creating a config file.
Essentially you'd have secrets leaking into your bash history, which is one of the goals we want to avoid.

Reply:
Clear, so I think we could:
1. write the file without encryption so the user can create it
1. or prompt for user input to get the value of the arg (so it will not appear in the history)
1. or other ideas?

Reply:
Committed the version with prompt: may continue on this?

Example:
```
$ node lib/cli.js config --arg ciao -v info
√ What is the value for ciao? · hello
[1564151059008] INFO : Saved ciao to ....\AppData\Roaming\releasify-nodejs\Config\releasify.json
```

The file is not readable by human:
```
Y7���v�{����k>�i�4E�Z�3P���U�%�(�*ـ�@��R���
```